### ISSUE TITLE
Finetune models

### ISSUE THREAD

**Issue by GeeYangML** [MLMonkATGY/DLDataPipeline#19]
1. Remove all aug
2. Increase model size
3. Add pos weight for focal loss
4. Train longer to increase conf
5. Freeze feature layers to accelerate training
7. Update optimizer to model classifier only

## Force tp  to be high
1. Increase pos weight to 30x
2. Change positive threshold to 0.3
